export default function handleError(err) {
  console.error('Caught error: ' + err);
}
